pgex/examples/glow_effect/main.py

```diff
@@ -0,0 +1,93 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+import asyncio
+import math
+from typing import Tuple
+
+import pygame
+
+Point = Tuple[int, int]
```

Comment:
This could go in a separate file like `_types.py` and be imported from there, then also no need for the `typing` import here, also change to
```py
Point: TypeAlias = Union[Tuple[int, int], List[int], pygame.Vector2]
```
could also add `Sequence[int]` to that union I guess

Reply:
I would put `Sequence[int]` directly instead of `List[int]`.

Because anyway `List` is derived from `Sequence`, and the problem with `List` is that it is mutable, comparing to `Tuple` or `pygame.Vector2`.

`List` would still be valid for the analyzer, but I guess it could be better to not explicitly show it.